speed_limiter path
speed_limiter shadow
speed_limiter obstacle
speed_limiter external